boxoffice/views/participant.py

```diff
@@ -34,5 +40,6 @@ def assign(order):
         assignee = Assignee(current=True, email=assignee_dict.get('email'), fullname=assignee_dict.get('fullname'),
         phone=assignee_dict.get('phone'), details=assignee_details, line_item=line_item)
         db.session.add(assignee)
+        boxofficeq.enqueue(send_ticket_assignment_email, line_item.id)
```

Comment:
Don't use `enqueue`. Add an `@job('boxoffice')` decorator to the function and use `send_ticket_assignment_email.delay(...)` here.